src/Archives/AdaptiveHuffmanTree.h

```diff
@@ -29,20 +32,21 @@ namespace Archives
 		// code and places the path in bitString
 
 	private:
-		void SwapNodes(USHORT node1, USHORT node2);
+		void VerifyValidNodeIndex(NodeIndex nodeIndex);
+		void SwapNodes(NodeIndex nodeIndex1, NodeIndex nodeIndex2);
 
+		// Tree properties
+		NodeType m_NumTerminalNodes;
+		NodeType m_NumNodes;
```

Comment:
I would prefer the name m_NodeCount here as well. Although not a big deal either way.

Reply:
Alright, I'll update the names. On a related matter, I think I would prefer dropping the "m_" prefix as well.